coalib/processes/BearRunning.py

```diff
@@ -246,11 +247,11 @@ def run_global_bear(message_queue,
 
 def run_local_bears_on_file(message_queue,
                             timeout,
-                            file_dict,
```

Comment:
To be backwards compatible, or at least break cleanly, `file_dict` here should be replaced with `file_contents`, and the function should check that `file_contents` is of the new expected type.